addons/project_todo/static/src/components/todo_chatter_panel/todo_chatter_panel.scss

```diff
@@ -1,12 +1,12 @@
 // Reduce horizontal spacing if the next sibling is an aside chatter
 .o_form_sheet_bg:has(+ .o_todo_chatter) {
-    @media screen and (max-width: 1920px) {
```

Comment:
Hey @stefanorigano,

By any chance, do you have an idea why (the f*) they used that breakpoint ? :thinking: